### DISCUSSION TITLE
Posters with parantheses in the name are not displayed

### DISCUSSION THREAD

**Discussion by seadeep42** [google/model-viewer#3879] [Q&A]
### Description
When a poster filename has `(` or `)`, it's not displayed.
I tested it with multiple files and it does seem to be the case.

### Live Demo
https://codepen.io/cdeep/pen/xxjMYRN
Two viewers side-by-side with only the poster name different.

### Version
Latest (2.0.2)

### Browser and OS
Chrome 105.0.5195.125 on macOS

**Comment by elalish** [comment]
This is a general web thing: URLs can't have parentheses in them. Your file name is actually a URL path to that file on your server.
